src/views/NoteView.vue

```diff
@@ -23,50 +21,69 @@
 </template>
 
 <script setup lang="ts">
-/*
-import{ref, onMounted} from 'vue';
-import { useRoute } from 'vue-router';
-
-interface Note {
-  revision: string;,
-  channel: string;,
-  permission: string;,
-  body: string;
-}
+import { NoteSummary } from '@/types/api';
+import { ref, computed } from 'vue';
+
```

Comment:
必要な依存関係の import をお願いします！

```suggestion
import { marked } from 'marked';
import DOMPurify from 'dompurify';

```